missions/a8/transcripts/_meta

```diff
@@ -1,82 +1,406 @@
 {
     "name": "a8",
-    "featured": false,
-    "incomplete": true,
+    "featured": true,
+    "incomplete": false,
```

Comment:
We could revert 5bf7265e6df518952809d9b83d8387a6cefc0419, and deploy the rest of the branch?